packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3142,6 +3390,131 @@ curl -v -X POST \
+
+#### Request Example for Verify WebAuthn Factor
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/fwf2rovRxogXJ0nDy0g4/verify"
+```
+
+#### Response Example for Verify WebAuthn Factor
+
+```json
+{
+  "factorResult":"CHALLENGE",
+  "profile":{
+    "credentialId":"l3Br0n-7H3g047NqESqJynFtIgf3Ix9OfaRoNwLoloso99Xl2zS_O7EXUkmPeAIzTVtEL4dYjicJWBz7NpqhGA",
+    "authenticatorName":"MacBook Touch ID"
+  },
+  "_links":{
+    "verify":{
+      "href":"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/fwf2rovRxogXJ0nDy0g4/verify",
+      "hints":{
+        "allow":[
+          "POST"
+        ]
+      }
+    },
+    "factor":{
+      "href":"https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/fwf2rovRxogXJ0nDy0g4",
+      "hints":{
+        "allow":[
+          "GET",
+          "DELETE"
+        ]
+      }
+    }
+  },
+  "_embedded":{
+    "challenge":{
+      "challenge":"vQFwTt6zKzMV7HFPzjS2",
+       "extensions":{
+       }
+    }
+  }
+}
+```
+
+#### Get the Signed Assertion from the WebAuthn Authenticator by calling the WebAuthn Javascript API
+
+```html
+<!-- Using CryptoUtil.js from https://github.com/okta/okta-signin-widget/blob/master/src/util/CryptoUtil.js -->
+<script>
+  // Convert activation object's challenge nonce from string to binary
+  response._embedded.challenge.challenge = CryptoUtil.strToBin(response._embedded.challenge.challenge); 
+
+  // Call the WebAuthn javascript API to get signed assertion from the WebAuthn authenticator
+  navigator.credentials.get({
+    publicKey: response._embedded.challenge
+  })
+    .then(function (assertion) {
+      // Get the client data, authenticator data, and signature data from callback result, convert from binary to string
+      var clientData = CryptoUtil.binToStr(assertion.response.clientDataJSON);
+      var authenticatorData = CryptoUtil.binToStr(assertion.response.authenticatorData);
+      var signatureData = CryptoUtil.binToStr(assertion.response.signature);
+    })
+    .fail(function (error) {
+      // Error from WebAuthn platform
+    });
+</script>
+```
+
+#### Post the Signed Assertion to Okta to Complete Verification
+
+#### Request Parameters for Verify WebAuthn Factor
+
+
+| Parameter      | Description                                         | Param Type | DataType | Required |
+| -------------- | --------------------------------------------------- | ---------- | -------- | -------- |
+| factorId       | `id` of factor returned from enrollment             | URL        | String   | TRUE     |
+| stateToken     | [state token](#state-token) for current transaction | Body       | String   | TRUE     |
```

Comment:
adding the word the

[state token](#state-token) for the current transaction

Reply:
I removed this entry (even for U2F, which was already here), as that must have been added in mistake -- there's no state token used in Factors API requests.